core/src/main/scala/kafka/controller/ControllerChannelManager.scala

```diff
@@ -39,7 +39,7 @@ import org.apache.kafka.common.{KafkaException, Node, TopicPartition}
 import scala.collection.JavaConverters._
 import scala.collection.mutable.HashMap
 import scala.collection.{Set, mutable}
-import org.apache.kafka.common.config.ClientDnsLookup
+import org.apache.kafka.clients.ClientDnsLookup
```

Comment:
There is a wildcard import in this class for `org.apache.kafka.clients`, so this import is not required.